packages/gestalt/src/contexts/SideNavigationProvider.js

```diff
@@ -33,8 +42,14 @@ type Props = {
     onDismiss: () => void,
     id: string,
   },
+  collapsible?: boolean,
+  collapsed?: boolean,
+  onCollapse?: (boolean) => void,
+  onPreview?: (boolean) => void,
 };
 
+const COLLAPSE_TRANSITION_DURATION = 200; // .2s
```

Comment:
We have to somehow keep this value consistent with the transition rule in `SideNavigation.css`

Reply:
Would this make sense to put in our tokens library and import it from there?